Automattic/wp-calypso
# Stats: Some styling adjustments per Odyssey design

#### Proposed Changes

- Changes the background color for "7-day highlights", "totals" "Year in review" per design
- Adds class for different sections for easier future CSS customzation
- Aligns Jetpack header with navigation menu
- Fixes incorrect color for dropdown navigation menu

#### Testing Instructions

* Setup Odyssey with Option 1 in `PejTkB-3E-p2`
* Open `/wp-admin/admin.php?page=stats`
* Ensure "7-day highlights" background color is `#f9f9f6`
<img width="868" alt="image" src="https://user-images.githubusercontent.com/1425433/215639098-3d68b5e2-42eb-41de-9176-32fee18966f1.png">

* Open Insights tab
* Ensure "2022 in review" background color is `#f9f9f6`
<img width="869" alt="image" src="https://user-images.githubusercontent.com/1425433/215639178-35ee782b-1843-4b5a-8e9e-1644893633c2.png">

* Open Ads tab - open `/wp-admin/admin.php?page=stats&flags=stats%2Fads-page`
* Ensure "Totals" background color is `#f9f9f6`

<img width="859" alt="image" src="https://user-images.githubusercontent.com/1425433/215639498-381b9ec5-afc2-4365-ab31-696345e5868a.png">

* Ensure other parts of the pages are not affected

* Change width of the screen
* Ensure Jetpack header and navigation are always aligned

* Change screen with to <480px
* Ensure Navigation items have the right color/background and hovering effects
<img width="437" alt="image" src="https://user-images.githubusercontent.com/1425433/215639819-7a65990b-4f3f-4cdf-bf84-d1349644fbab.png">


#### Pre-merge Checklist





Related to #
